### PR TITLE
Add Polymorphisation paper

### PR DESCRIPTION
**Pull request by aDotInTheVoid**

### CODE REVIEW

File: src/appendix/bibliography.md [hunk 1]
```diff
@@ -89,3 +89,4 @@ Rust, as well as publications about Rust.
 * [The Case for Writing a Kernel in Rust](https://www.tockos.org/assets/papers/rust-kernel-apsys2017.pdf)
 * [RustBelt: Securing the Foundations of the Rust Programming Language](https://plv.mpi-sws.org/rustbelt/popl18/)
 * [Oxide: The Essence of Rust](https://arxiv.org/abs/1903.00982). By Aaron Weiss, Olek Gierczak, Daniel Patterson, Nicholas D. Matsakis, and Amal Ahmed.
+* [Polymorphisation: Improving Rust compilation timesthrough intelligent monomorphisation](https://davidtw.co/media/masters_dissertation.pdf)/ By David Wood
```
Comment (JohnTitor) [inline src/appendix/bibliography.md:92]: nit:

```suggestion
* [Polymorphisation: Improving Rust compilation timesthrough intelligent monomorphisation](https://davidtw.co/media/masters_dissertation.pdf). David Wood's master's thesis.
```

**Review by JohnTitor — APPROVED** [review]
We already mention it in https://rustc-dev-guide.rust-lang.org/backend/monomorph.html#polymorphization but it makes sense that we also have it in the bibliography 👍

File: src/appendix/bibliography.md (hunk 1 above)
Comment (camelid) [reply src/appendix/bibliography.md:92]: Also, there's a missing space between `times` and `through`.

**Comment by camelid** [comment]
@aDotInTheVoid could you address the review comments? Thanks!

**Comment by aDotInTheVoid** [comment]
Done, sorry for the wait

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**